esp-hal-common/devices/esp32.toml

```diff
@@ -0,0 +1,18 @@
+[device]
+arch  = "xtensa"
+cores = "multi_core"
```

Comment:
nitpick: cores should probably be an integer, then we can output `"single_core"` or `"multi_core"` based on that integer in the build script

Reply:
I considered that, but does it really matter? With that approach we're basically encoding data in an integer (core count) here and then throwing that data away anyway, so what purpose does that number actually serve?